.circleci/config.yml

```diff
@@ -202,7 +202,7 @@ jobs:
 
   iOS build and test:
     macos:
-      xcode: "11.2.0"
+      xcode: "11.2.1"
```

Comment:
Does this have implications for the generated code? Would this require apps to use the same version we specify here?

Reply:
Nope, it's the same image: https://circle-macos-docs.s3.amazonaws.com/image-manifest/v1532/index.html
_Actual_ Xcode 11.2.0 was pulled, because it was next to unusable (contained code generator bugs, Apple itself discourages use of that).
CircleCI replaced 11.2.0 with 11.2.1 on its own.
I only updated that here to reflect better that we're using 11.2.1

Reply:
Thanks for clarifying. Let's give this a couple of more shots on CI before merging.